cpp/src/parquet/column_reader.h

```diff
@@ -115,11 +116,30 @@ class PARQUET_EXPORT PageReader {
                                           bool always_compressed = false,
                                           const CryptoContext* ctx = NULLPTR);
 
+  // If skip_page_callback_ is present (not null), NextPage() will call the
+  // callback function exactly once per page in the order the pages appear in
+  // the column. If the callback function returns true the page will be
+  // skipped. The callback will be called only if the page type is DATA_PAGE or
+  // DATA_PAGE_V2. Dictionary pages will not be skipped.
+  // This setter must be called at most once to set the callback.
+  // \note API EXPERIMENTAL
+  void set_skip_page_callback(
+      std::function<bool(const DataPageStats&)> skip_page_callback) {
```

Comment:
Can we define a type name for this? For example:
```c++
  using DataPageFilter = std::function<bool(const DataPageStats&)>;
```

Reply:
Done.